locales/it/index.yml

```
@@ -550,6 +550,7 @@ wallet:
     titlePagoPaUpdateApp: Aggiornamento richiesto
     messagePagoPaUpdateApp: "IO introduce spesso piccole migliorie e nuove funzioni: per continuare ad usare il portafoglio è necessario aggiornare l'applicazione all'ultima versione."
     btnUpdateApp: Aggiorna l'app IO
+    favourite: Non è possibile rimuovere un metodo di pagamento preferito, per modificarlo selezionane un altro
```

```suggestion
    favourite: Il metodo selezionato, essendo l'unico, è anche il preferito. Per impostare un altro metodo preferito aggiungilo e fai tap sulla stellina.
```
Could it be more clear?